src/Symfony/Component/Serializer/Tests/Fixtures/IntegerBackedEnumDummy.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+
+namespace Symfony\Component\Serializer\Tests\Fixtures;
+
+
+enum IntegerBackedEnumDummy: int
+{
+    case SUCCESS = 200;
```

Comment:
For test one value is enough isn't it? Let's save some resources 

Same for the others